### PR TITLE
Closes #1

### LINKED ISSUE #1: HTML Markup and styling according to a wireframe
- [ ]  HTML markup 
- [ ] CSS styling

### PR DESCRIPTION
**Pull request by pavelbidenko2018**

### CODE REVIEW

**Review by okozmovskaya — CHANGES_REQUESTED** [review]
Cool css style! I've never meet before the scheme with link to color in css, it's very interesting)))
Just need to add wireframe and changes about your step in dev-strategy.

**Review by okozmovskaya — APPROVED** [review]

**Review by oguzkarademir — CHANGES_REQUESTED** [review]
can you add your user story and wireframe to development stretegy @pavelbidenko2018

File: index.html [hunk 1]
```diff
@@ -2,47 +2,61 @@
 <html lang='en'>
 
 <head>
-  <meta charset='utf8'>
-  <title>Arraying</title>
+    <meta charset='utf8'>
```
Comment (Sayed94h) [inline index.html]: you should change the _charset='utf8'_ to _charset='utf-8'_ because if you validate the html later then you will get error.

**Review by Sayed94h — COMMENTED** [review]
I reviewed your codes. I added a comment in your html code.

**Review by okozmovskaya — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (pavelbidenko2018) [reply index.html]: Thank you

**Review by oguzkarademir — APPROVED** [review]

**Pull request merged**